components/bl-accordion-component/README.md

```diff
@@ -0,0 +1,58 @@
+# Accordion
+
+Accordion is a component of Backendless UI-Builder designer. It allows the user to show and hide sections of related content on a page. The component supports basic and controlled type of accordion.
+
+Customizations include adjustments of font size, color, background color and padding for both title and content.
+
+## Usage
+
+After adding the component to the page, specify the accordion data. This can be done in UI-Builder designer or using codeless logic.
+
+### Styles and Settings
+
+<dl>
+<dt>Accordion Data</dt>
+<dd>Array of objects that define the title and content of the accordion sections. Can be determined by Accordion Data Logic.</dd>
+<dt>Controlled Accordion</dt>
+<dd>Checkbox for determining whether multiple accordion sections can be expanded at the same time. If property returns true, only one section content can be visible at a time. Unchecked by default.</dd>
+<dt>Title Font Size</dt>
+<dd>Value from this field will be used as the font size of the accordion section title.</dd>
+<dt>Title Color</dt>
+<dd>Color used for the accordion section title. Default depends on the selected theme.</dd>
+<dt>Title Background Color</dt>
+<dd>Color used as the background of the accordion section title. Default depends on the selected theme.</dd>
+<dt>Title Padding</dt>
+<dd>Value from this field will be used as the padding for the accordion section title.</dd>
+<dt>Content Font Size</dt>
+<dd>Value from this field will be used as the font size of the accordion section content.</dd>
+<dt>Content Color</dt>
+<dd>Color used for the accordion section content. Default depends on the selected theme.</dd>
+<dt>Content Background Color</dt>
+<dd>Color used as the background of the accordion section content. Default depends on the selected theme.</dd>
+<dt>Content Padding</dt>
+<dd>Value from this field will be used as the padding for the accordion section content.</dd>
+</dl>
+
+### Event Handlers and Bindable Properties
+
+<dl>
+<dt>Accordion Data Logic</dt>
+<dd>This is the handler for the logic to determine the title and content of the accordion sections. Array of objects returned by the handler will define the data for the accordion.</dd>
+<dt>On Click Event</dt>
```

Comment:
On Item Click Event? Also, from description: "clicks the mouse or taps the title". What if we click on content?

Reply:
This event is fired only when we open/close some section. This can be done by clicking on the title. So if we click on the content nothing happens. Do you think it is better to move this event to the item?

Reply:
I see two variants

1) Rename it to On Open/Close Item Event

2) Or make it possible to trigger events no matter where we click - on the content or title.

Reply:
On Toggle Item Event ?